src/sonic_ax_impl/mibs/ietf/rfc4363.py

```diff
@@ -0,0 +1,68 @@
+import json
+from enum import unique, Enum
+
+from sonic_ax_impl import mibs
+from sonic_ax_impl.mibs.ietf import *
```

Comment:
I don't recommend `import *` in any production code. For the mac_decimals function (see my other comment) and:

```python
from ax_interface import ..., util
```